src/framework/preferences/js/Panels.js

```diff
@@ -841,7 +841,7 @@ var fluid_2_0 = fluid_2_0 || {};
             theme: ["default", "bw", "wb", "by", "yb", "lgdg"]
         },
         markup: {
-            label: "<span class=\"fl-preview-A\">A</span><span class=\"fl-hidden-accessible\">%theme</span><div class=\"fl-crossout\"></div>"
+            label: "<span class=\"fl-preview-A\"></span><span class=\"fl-hidden-accessible\">%theme</span><div class=\"fl-crossout\"></div>"
```

Comment:
@jobara It's likely due to the accessible name computation that browsers implement.  As a first approximation:  include text specified in CSS content rules.  The details are here:
http://w3c.github.io/aria/accname-aam/accname-aam.html#step2F, especially bullet ii.

Thus, it's not necessarily that the screen reader is picking it up; rather, the browser is telling the screen reader:  the name is, "...", and the "..." includes CSS text content.